plugins/Ecoji/__init__.py

```diff
@@ -0,0 +1,18 @@
+import discord
+import requests
+import urllib
+import json
+from discord.ext import commands
+
+
+from Plugin import AutomataPlugin
+
+class Ecoji(AutomataPlugin):
+    """Sends randomly generated emojis from Jack Harrhy's Ecoji project"""
+    @commands.command()
+    async def ecoji(self, ctx: commands.Context, limit):
+        ecojiSrc = "https://jackharrhy.dev/urandom/ecoji/"+str(limit)
+        ecojiTxt = urllib.request.urlopen(ecojiSrc)
```

Comment:
preferably instead of using urllib directly, you'd use https://docs.python-requests.org/en/master/, something we're already using in places like: https://github.com/MUNComputerScienceSociety/Automata/blob/0cd136abf6ddfa9d180eff6c0e621ae0546b246e/plugins/ExecutiveDocs/__init__.py#L50-L51